eventqueue-compose/src/main/kotlin/ObserveEvents.kt

```diff
@@ -0,0 +1,41 @@
+package com.redmadrobot.eventqueue.compose
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.platform.LocalLifecycleOwner
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.flowWithLifecycle
+import com.redmadrobot.eventqueue.Event
+import com.redmadrobot.eventqueue.flow.EventQueue
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+
+/**
+ * Observes events from the provided [EventQueue] and triggers the specified action when new events occur.
+ * @param events The EventQueue from which events will be observed
+ * @param lifecycleOwner The LifecycleOwner to use for observing lifecycle changes (default is the current LocalLifecycleOwner)
+ * @param minActiveState The minimum active state of the LifecycleOwner for observing events (default is Lifecycle.State.STARTED)
+ * @param onEvent The action to perform when a new event is observed, defined as a lambda expression taking an Event as a parameter
+ */
+@Suppress("ComposableParametersOrdering", "FunctionNaming")
+@Composable
+public fun ObserveEvents(
+    events: EventQueue,
+    lifecycleOwner: LifecycleOwner? = LocalLifecycleOwner.current,
+    minActiveState: Lifecycle.State = Lifecycle.State.STARTED,
+    onEvent: CoroutineScope.(Event) -> Unit,
```

Comment:
Maybe it'll be better to make this lambda suspend and remove scope from receiver.
```suggestion
    onEvent: suspend (Event) -> Unit,
```